tests/suite/custom_resources_utils.py

```diff
@@ -260,15 +260,31 @@ def create_virtual_server_from_yaml(
     print("Create a VirtualServer:")
     with open(yaml_manifest) as f:
         dep = yaml.safe_load(f)
+
+    return create_virtual_server(custom_objects, dep, namespace)
+
+
+def create_virtual_server(
```

Comment:
there's already a function called `create_virtual_server_from_yaml` so no need to read yaml and pass it to a new fucntion.